sdcm/nemesis.py

```diff
@@ -991,6 +1019,13 @@ def disrupt(self):
         self.disrupt_nodetool_cleanup()
 
 
+class TruncateMonkey(Nemesis):
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.disrupt_truncate()
+
+
```

Comment:
Add also to the list of LimitedChaosMonkey

Reply:
done